packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx

```diff
@@ -31,21 +31,21 @@ export const FlowContainer = ({
     : ftlMsgResolver.getMsg('flow-container-back', 'Back');
   return (
     <div
-      className="max-w-lg mx-auto mt-6 p-10 tablet:my-10 flex flex-col items-start bg-white shadow tablet:rounded-xl"
+      className="max-w-lg mx-auto mt-6 p-10 mobileLandscape:my-10 flex flex-col items-start bg-white shadow mobileLandscape:rounded-xl"
       data-testid="flow-container"
     >
       <Head title={title} />
 
-      <div className="flex items-center">
+      <div className="relative flex items-center">
         <button
           onClick={onBackButtonClick}
           data-testid="flow-container-back-btn"
           title={backButtonTitle}
-          className="relative w-8 h-8 -ms-2 me-2 tablet:me-10 tablet:-ms-18"
+          className="me-4 mobileLandscape:absolute mobileLandscape:p-4 mobileLandscape:-start-24"
         >
-          <BackArrow className="w-8 h-auto pe-2 absolute top-1/2 start-1/4 transform -translate-y-1/2 ltr:-translate-x-1/2 rtl:translate-x-1/2 rtl:-scale-x-100 fill-current" />
+          <BackArrow className="w-6 h-auto fill-grey-400 rtl:transform rtl:-scale-x-100" />
         </button>
-        <h1 className="font-header text-lg mb-1">{title}</h1>
```

Comment:
Was removing `mb-1` intentional? I'm wondering if that affects any other layouts using this, or if our other headers had a `margin` anyway that was collapsed.

Reply:
It was intentional, but it would make sense to keep the margin if/when there is a sub-heading. I'll move the margin to the h2 element instead.